src/app/pages/StoryPage/index.jsx

```diff
@@ -256,18 +268,22 @@ const StoryPage = ({ pageData, mostReadEndpointOverride }) => {
           columns={gridColsSecondary}
           parentColumns={gridColumns}
         >
-          <ResponsiveComponentWrapper>
-            <TopStories
-              content={topStoriesInitialData}
-              parentColumns={gridColsSecondary}
-            />
-          </ResponsiveComponentWrapper>
-          <ResponsiveComponentWrapper>
-            <FeaturesAnalysis
-              content={featuresInitialData}
-              parentColumns={gridColsSecondary}
-            />
-          </ResponsiveComponentWrapper>
+          {topStoriesInitialData && (
```

Comment:
Just an after-thought from the first review. Do we want to be doing the suppression on page level or would it be better to do the suppression at the container level? Not sure what the general practice is for this.

Reply:
I agree with @HarveyPeachey, we've got conditional statements in several containers where we `return null` if certain conditions are not met.

Reply:
I think we're already suppressing [at the container level](https://github.com/bbc/simorgh/blob/latest/src/app/containers/CpsOnwardJourney/index.jsx#L122), but just doing that doesn't stop the the [padding/margin in the responsive wrapper](https://github.com/bbc/simorgh/blob/latest/src/app/pages/StoryPage/index.jsx#L138-L143) from being applied